### ISSUE TITLE
Add OPRM URLs to proxy list for header injection

### ISSUE THREAD

**Issue by batemapf** [department-of-veterans-affairs/va.gov-team#658]
Not sure what the process here is, but OPRM has requested that we add the following:

`www.oprm.va.gov`
`vaww.oprm.webdeve.va.gov`

The OPRM team will take care of QA and any fixes on their end.

**Comment by rianfowler** [comment]
# Tasks
- [ ] request update to <head> tag on the Teamsite template used by www.oprm.va.gov. Contingent on Josh Tuscher's (Joshua.Tuscher@va.gov) or new admin's availability. (it's been a few months since we've requested an update). 
- [ ] add the urls to the allow list behind the cookie only flag. This will enable testing on production behind a flag controlled by a cookie setting. 
- [ ] test, fix, verify CSS issues. CSS overrides should be added [here](https://github.com/department-of-veterans-affairs/vets-website/tree/master/src/applications/proxy-rewrite/sass).

# Notes
- See [documentation](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/platform/architecture/teamsite/)
- Extensive CSS fixes may require some VSP support. It's possible to run the site locally to build the CSS assets and point the production site to the locally hosted version of those assets. This process has not be extensively tested / used.

**Comment by JeffBarnesUSDS** [comment]
@rianfowler please let me know when testing is complete and you are ready to remove the cookie only flag so I can coordinate with the business owner. thanks!

**Comment by rianfowler** [comment]
@JeffBarnesUSDS @batemapf  do we have a point of contact for Teamsite administration. I tried to email Joshua.Tuscher@va.gov yesterday to work out a time to work together on adding in the scripts to the page / template but the message was returned as undeliverable.

**Comment by JeffBarnesUSDS** [comment]
Tuscher, Joshua <Joshua.Tuscher@va.gov> is the correct email. Not sure what happened. He is the best POC for this.

**Comment by rianfowler** [comment]
This has been handed off to Josh Tuscher for testing. I fixed a few CSS issues and provided these instructions for testing. 

```
1. Navigate to https://www.oprm.va.gov/
2. Open the javascript console (F12 in Chrome or IE -> open the “console” tab)
3. Paste this into the console and press return: 

document.cookie = "proxyRewrite=true;”

4. Close the javascript console (press F12 again, or click the x on the pane)
5. Refresh the page
```

I think we're going to track follow-up work in a separate issue for the next sprint. @brandonrapp can confirm.
